azure-pipelines.yml

```diff
@@ -159,6 +163,17 @@ jobs:
       $env:PATH = "C:\\ProgramData\\chocolatey\\lib\\mingw\\tools\\install\\mingw$(BITS)\\bin;" + $env:PATH
 
       mkdir dist
+
+      If ( Test-Path env:NPY_USE_BLAS_ILP64 ){
+        #echo "##vso[task.setvariable variable=OPENBLAS64_]$pwd\openblaslib"
+        $env:OPENBLAS64_ = "$pwd\openblaslib"
+      } else {
+        #echo "##vso[task.setvariable variable=OPENBLAS]$pwd\openblaslib"
+        $env:OPENBLAS = "$pwd\openblaslib"
+      }
+      Write-Host "::: $env:OPENBLAS"
+      Write-Host "::: $env:OPENBLAS64_"
+
```

Comment:
In order to debug what was going on I tried this on linux (after downloading the openblas tarball and expanding it in `/tmp/local/`):
```
OPENBLAS64_=/tmp/local python -c \
"import builtins; builtins.__SCIPY_SETUP__=1; \
    from numpy.distutils.system_info \
    import get_info; print(get_info('xxxx', 2))"
```

I am not sure what `xxxx` should be in scipy, the `setup.py` is different

Reply:
If the environment variables are a suitable replacement for the `site.cfg` approach below, then presumably both are needed for 64-bit int case while only the one for 32-bit int case. That might be a more concise modification.